incubation-process.md

```diff
@@ -87,19 +87,15 @@ Projects at the Adopted stage fit into the overall architecture of the OpenRail
 
 ### Criteria
 
-*The criteria for stage 3 are still a draft*
-
-*Examples from CNCF incubation process, to be adopted*
-
 * Document that it is being used successfully in production by at least three independent adopters which, in the TC’s judgement, are of adequate quality and scope.
+* Have a public list of project adopters for at least the primary repo (preferably documented in the project's publiccode.yml file). For a specification, have a list of adopters for the implementation(s) of the spec.
 * Have committers from at least two organizations.
-* Have achieved and maintained an OpenSSF Best Practices Badge.
-* Provide SBOMs for all release artifacts
+* The project has a documented track record of regular stable releases
+* The project has a documented process with guaranteed reaction times for releasing security updates
 * Have completed an independent and third party security audit with results published of similar scope and quality as this example which includes all critical vulnerabilities and all critical vulnerabilities need to be addressed before moving to the Adopted stage.
```

Comment:
This is a one-time requirement. We need a process for an ongoing check and possibly audits of security issues.

The audit should also cover more general questions such as architecture from a security point of view and what are the implications of working on it and maintaining it in the future in a secure way.